src/CMakeLists.txt

```diff
@@ -5,7 +5,19 @@
 # the ROCm hipBLAS implementation, we use the ROCm
 # hipBLAS library's hipblas.h header.
 include(ExternalProject)
-option(H4I_ROCM_HIPBLAS_TAG "Tag to use from the ROCm hipBLAS repository when obtaining its hipblas.h header" "rocm-5.3.0")
+# make sure that HIP_VERSION_MAJOR, HIP_VERSION_MINOR, and HIP_VERSION_PATCH are defined
+if(NOT DEFINED HIP_VERSION_MAJOR)
```

Comment:
HIP_VERSION_MAJOR, MINOR or PATCH will NOT be defined in most of the cases. May I know from where we will get them? 
I think either submodule approach or manually updating the Tag ("rocm-5.3.0" to "rocm-6.1.1") is the best we can do.

Reply:
> HIP_VERSION_MAJOR, MINOR or PATCH will NOT be defined in most of the cases. May I know from where we will get them?

Can you give me a scenario where this would be true?

in either case I am fine with hardcoding 6.1.1 for now.

Reply:
Great! lets change "rocm-5.3.0" to latest rocm. I will verify once done and merge.